### PR TITLE
DEV: Add missing import, de-jQuerify

### PR DESCRIPTION
**Pull request by CvX**

### CODE REVIEW

File: app/assets/javascripts/discourse/tests/acceptance/group-manage-email-settings-test.js [hunk 1]
```diff
@@ -341,7 +341,7 @@ acceptance(
 
 //     needs.pretender((server, helper) => {
 //       server.post("/groups/47/test_email_settings", () => {
-//         return helper.response(400, {
```
Comment (eviltrout) [inline app/assets/javascripts/discourse/tests/acceptance/group-manage-email-settings-test.js:344]: Mostly I like this commit very much, but I'm curious why you updated code in comments? Seems peculiar.

Comment (CvX) [reply app/assets/javascripts/discourse/tests/acceptance/group-manage-email-settings-test.js]: The second one will be useful when I uncomment that test. This one, I'm reverting (though will probably change in that separate, second PR 🙂)

**Review by martin-brennan — APPROVED** [review]
This is great thanks, but why don't our automated tools nag me if I don't import `exists`? They do if I don't import `query`/`queryAll`

**Comment by CvX** [comment]
@martin-brennan `exists` is on our allowed globals list: https://github.com/discourse/eslint-config-discourse/blob/34c8d5f9f36b3b8b2e46d7f7c4ca68cc0868efe5/index.js#L40

Eventually we'd like to remove it, but doing so would break many plugins' builds, so first it'd be good to have a system that would fix those issues automatically.

**Pull request merged**